sensor_msgs/Image image
---
orvis/ImageMasks objects